packages/runner/src/errors/script-error.spec.jsx

```diff
@@ -5,9 +5,21 @@ import ScriptError from './script-error'
 
 describe('<ScriptError />', () => {
   it('renders nothing when there is no script error', () => {
-    const state = { scriptError: null }
-    const component = shallow(<ScriptError state={state} />)
+    const state = { error: null }
+    const component = shallow(<ScriptError {...state} />)
 
     expect(component).to.be.empty
   })
+
+  it('renders ansi as colors', () => {
```

Comment:
i can't seem to find this test in the output of the `unit-tests` stage... are these tests running? :/ https://circleci.com/gh/cypress-io/cypress/272659?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
The test script was broken since it didn't have a glob of files to run. It just ran 0 tests and passed. I've fixed it.